CHANGELOG.md

```diff
@@ -64,9 +64,9 @@ Our versioning strategy is as follows:
 * `[sitecore-jss]` GenericFieldValue model is updated to accept Date type ([#1916](https://github.com/Sitecore/jss/pull/1916))
 * `[template/node-xmcloud-proxy]` `[sitecore-jss-proxy]` Introduced /api/healthz endpoint ([#1928](https://github.com/Sitecore/jss/pull/1928))
 * `[sitecore-jss]` `[sitecore-jss-angular]` Render field metdata chromes in editMode metadata - in edit mode metadata in Pages, angular package field directives will render wrapping `code` elements with field metadata required for editing; ([#1926](https://github.com/Sitecore/jss/pull/1926))
-* `[sitecore-jss-nextjs]` Expose MiddlewareBase class and MiddlewareBaseConfig type ([#1941](https://github.com/Sitecore/jss/pull/1941))
 * `[angular-xmcloud]``[sitecore-jss-angular]` Analytics and CloudSDK integration
-* `[angular-xmcloud]` Add CloudSDK initialization on client side ([#1952](https://github.com/Sitecore/jss/pull/1952))
+  * `[angular-xmcloud]` Add CloudSDK initialization on client side ([#1952](https://github.com/Sitecore/jss/pull/1952))
+  * `[angular-xmcloud]``[sitecore-jss-angular]` Add CDP page view component to angular xmc app and add it to the app's scripts section. ([#1957](https://github.com/Sitecore/jss/pull/1957))
```

Comment:
Do we need to add a migration guide entry? I think we need, since customers will need to integrate that

Reply:
hm do we need that? In this case the angular xmc app is new, so there wouldn't be anything to migrate/upgrade from, correct? or you mean we need to indicate the new feature to the customers in the changelog or upgrade guide?